plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt

```diff
@@ -58,9 +57,18 @@ class FeatureDevSessionContext(val project: Project) {
         return ZipCreationResult(zippedProject, checkSum256, zippedProject.length())
     }
 
-    fun ignoreFile(file: File): Boolean = ignorePatternsWithGitIgnore.any { p -> p.containsMatchIn(file.path) }
+    fun ignoreFile(file: File, addGitignore: Boolean = true): Boolean {
```

Comment:
This was discussed with our team and I need to pull back this requirement. You could go forward and just add the .gitignore pattern to the list. Sorry for the inconvenience.

Reply:
Added gitignore back.